labelbox/schema/bulk_import_request.py

```diff
@@ -113,6 +115,76 @@ class BulkImportRequest(DbObject):
     project = Relationship.ToOne("Project")
     created_by = Relationship.ToOne("User", False, "created_by")
 
+    @property
+    def inputs(self) -> List[Dict[str, Any]]:
+        """
+        Inputs for each individual annotation uploaded.
+        This should match the ndjson annotations that you have uploaded. 
+        
+        Returns:
+            Uploaded ndjson.
+
+        * This information will expire after 24 hours.    
+        """
```

Comment:
do you need a wait until done here?

Reply:
No this one is immediately available since it is created when the BulkImportRequest object is created.